Client/src/Components/Authentication/ResetPasswordView.tsx

```diff
@@ -0,0 +1,105 @@
+import { FastField, Field, Form, Formik } from 'formik'
+import { defaultResetPasswordModel, IResetPasswordModel } from '../../Models/Authentication/ISignUpModel'
+import React from 'react'
+
+import Avatar from '@material-ui/core/Avatar'
+import Button from '@material-ui/core/Button'
+import Container from '@material-ui/core/Container'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined'
+import { MuiTextFieldFormik } from '../Forms/FormikFields'
+import Typography from '@material-ui/core/Typography'
+import { callResetPassword } from '../../Remote/Endpoints/AuthenticationEndpoint'
+import { resetPasswordValidationSchema } from './AuthenticationValidationSchema'
+import { makeStyles } from '@material-ui/core/styles'
+import { useParams } from 'react-router'
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
```

Comment:
not a fan of make styles in place like this, imo we should find a way to make it reusable, there is an appTheme file in the root folder, or use the Grid/Box for material ui as much as possible for clean code

Reply:
thats for the future to keep in mind tho of course

Reply:
Alright sure, I just copied it from another modal view we had